package.json

```diff
@@ -84,5 +75,12 @@
         "lint": "eslint --cache clients/web Gruntfile.js grunt_tasks scripts clients/web/test && pug-lint clients/web/src/templates",
         "docs": "grunt docs",
         "esdoc": "esdoc"
+    },
+    "files": [
+        "clients/web/src/"
+    ],
+    "main": "clients/web/src/index.js",
```

Comment:
I can go either way on this. [Here](https://webpack.github.io/docs/resolving.html#resolving-an-absolute-path) are the docs of how the `main` field of package.json is used by webpack (which is the only way we are using this field, since neither of these makes sense as a node entry point). Obviously our index.js should be the `main` for girder/core, but I'm not sure how someone might even use `main` in the outer app package -- maybe it makes sense to simply omit it for now. What do you think?